code/modules/mob/living/simple_animal/abnormality/he/golden_false_apple.dm

```diff
@@ -127,10 +162,10 @@
 	H.update_light()
 	return ..()
 
-/datum/status_effect/stacking/golden_sheen/tick()//change this to golden apple's life tick for less lag
+/datum/status_effect/stacking/golden_sheen/tick()//TODO:change this to golden apple's life tick for less lag
 	var/mob/living/carbon/human/H = owner
-	H.adjustBruteLoss(stacks * -1)//heals much faster and roughly twice as much as child of the galaxy per stack, but only lasts a minute
-	H.adjustSanityLoss(stacks * -1)
+	H.adjustBruteLoss(stacks * -0.5)
+	H.adjustSanityLoss(stacks * -0.5)
```

Comment:
Now gets added to abnormalities and other hostile creatures, assuming human is no longer reasonable.